mslice/tests/workspace_test.py

```diff
@@ -0,0 +1,217 @@
+import unittest
```

Comment:
`PixelWorkspaceTest` definitely belongs in its own file, but I'm unsure about the others.

The tests for `Workspace` and `HistogramWorkspace` are basically the same, so they inherit from `BaseWorkspaceTest` (which doesn't actually have any tests on its own, just provides common implementation to the two subclasses). We might want to keep these in the same file since they're so closely related?

Reply:
@MatthewBowles I think it would still be good to split them up. If I'm looking for the `HistogramWorkspace` test then it's not that obvious that it's with the workspace one. Can you just import the `BaseWorkspaceTest` class in the `Histogram` test?

Reply:
Good point, will do.